### PR TITLE
add arxiv badge and link to torchtitan paper

### PR DESCRIPTION
**Pull request by lessw2020**
This PR:
1 - adds an arvix badge underneath the two CI badges that links to the titan arxiv paper.
2 - adds a two sentence section in the main body summarizing our paper with link
3 - makes a minor grammar correction regarding using latest nightly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +20,9 @@ Our guiding principles when building `torchtitan`:
 
 [![Welcome to torchtitan!](assets/images/titan_play_video.png)](https://youtu.be/ee5DOEqD35I?si=_B94PbVv0V5ZnNKE "Welcome to torchtitan!")
 
+### Our torchtitan paper on Arxiv:
```
Comment (tianyu-l) [inline README.md]:
```suggestion
### Our torchtitan paper on arXiv
```

File: README.md [hunk 2]
```diff
@@ -18,6 +20,9 @@ Our guiding principles when building `torchtitan`:
...
+We provide a detailed look into the parallelisms and optimizations available in titan, along with summary advice on when to use various techniques:  [TorchTitan: One-stop PyTorch native solution for production ready LLM pre-training](https://arxiv.org/abs/2410.06511)
```
Comment (tianyu-l) [inline README.md]:
```suggestion
We provide a detailed look into the parallelisms and optimizations available in `torchtitan`, along with summary advice on when to use various techniques:  [TorchTitan: One-stop PyTorch native solution for production ready LLM pre-training](https://arxiv.org/abs/2410.06511)
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 [![4 GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_4gpu.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_4gpu.yaml?query=branch%3Amain)
 [![8 GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu.yaml?query=branch%3Amain)
 
+[![arXiv](https://img.shields.io/badge/arXiv-2410.06511-b31b1b.svg?style=plastic)](https://arxiv.org/abs/2410.06511)
```
Comment (tianyu-l) [inline README.md]: The badge looks interesting. I personally prefer putting this badge right after the section title "Our torchtitan paper on arXiv". (e.g. the red coloring here looks like suggesting some test failure?)

File: README.md (hunk 1 above)
Comment (lessw2020) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (lessw2020) [reply README.md]: good catch, updated to ensure consistency.

File: README.md (hunk 3 above)
Comment (lessw2020) [reply README.md]: makes sense re: possible mis-interpret red as something failing.
I've moved to just under the section title.

**Review by tianyu-l — APPROVED** [review]
LGTM, thanks!

**Pull request merged**